src/index.js

```diff
@@ -84,17 +88,25 @@ class FBMessenger {
       cb = notificationType
       notificationType = this.notificationType
     }
+
+    const json = {
+      recipient: {
+        id: id
+      }
+    }
```

Comment:
not need in this case. [Automatic Semicolon Insertion](http://www.ecma-international.org/ecma-262/6.0/#sec-automatic-semicolon-insertion)